testsuite/src/it/java/org/graalvm/tests/integration/utils/Commands.java

```diff
@@ -68,28 +72,32 @@
 public class Commands {
 
     private static final Logger LOGGER = Logger.getLogger(Commands.class.getName());
-    public static final String CONTAINER_RUNTIME = getProperty(
-            new String[]{"QUARKUS_NATIVE_CONTAINER_RUNTIME", "quarkus.native.container-runtime"},
-            "docker");
-    public static final String BUILDER_IMAGE = getProperty(
-            new String[]{"QUARKUS_NATIVE_BUILDER_IMAGE", "quarkus.native.builder-image"},
-            "quay.io/quarkus/ubi-quarkus-mandrel:21.3-java11");
+
+    public static final String CONTAINER_RUNTIME = getProperty("QUARKUS_NATIVE_CONTAINER_RUNTIME", "docker");
     // Podman: Error: stats is not supported in rootless mode without cgroups v2
-    public static final boolean PODMAN_WITH_SUDO = Boolean.parseBoolean(
-            getProperty(new String[]{"PODMAN_WITH_SUDO", "podman.with.sudo"}, "true"));
-    public static final QuarkusVersion QUARKUS_VERSION = new QuarkusVersion(
-            getProperty(
-                    new String[]{"QUARKUS_VERSION", "quarkus.version"},
-                    "2.7.6.Final"));
-    public static final boolean FAIL_ON_PERF_REGRESSION = Boolean.parseBoolean(
-            getProperty(new String[]{"FAIL_ON_PERF_REGRESSION", "fail.on.perf.regression"}, "true"));
+    public static final boolean PODMAN_WITH_SUDO = Boolean.parseBoolean(getProperty("PODMAN_WITH_SUDO", "true"));
+    public static final boolean FAIL_ON_PERF_REGRESSION = Boolean.parseBoolean(getProperty("FAIL_ON_PERF_REGRESSION", "true"));
+
     public static final boolean IS_THIS_WINDOWS = System.getProperty("os.name").matches(".*[Ww]indows.*");
     private static final Pattern NUM_PATTERN = Pattern.compile("[ \t]*[0-9]+[ \t]*");
     private static final Pattern ALPHANUMERIC_FIRST = Pattern.compile("([a-z0-9]+).*");
     private static final Pattern CONTAINER_STATS_MEMORY = Pattern.compile("(?:table)?[ \t]*([0-9\\.]+)([a-zA-Z]+).*");
 
-    public static String getProperty(String[] alternatives, String defaultValue) {
+    public static final String GRAALVM_ALLOW_DEPRECATED_BUILDER_SWITCH_TOKEN = "<GRAALVM_ALLOW_DEPRECATED_BUILDER>";
+    public static final String GRAALVM_ALLOW_DEPRECATED_BUILDER_SWITCH = "-H:+AllowDeprecatedBuilderClassesOnImageClasspath";
```

Comment:
@zakkak So I guess I can get rid of this now...?

Reply:
Yes, but only for Quarkus >= 2.14. Which makes sense since 2.13 only supports GraalVM/Mandrel 22.3